src/schema/v2/artworksForUser/artworksForUser.ts

```diff
@@ -50,7 +52,16 @@ export const artworksForUser: GraphQLFieldConfig<void, ResolverContext> = {
       context
     )
 
-    const artworks = [...newForYouArtworks, ...backfillArtworks]
+    const allArtworks = [...newForYouArtworks, ...backfillArtworks]
+
+    // Because we load the data from two different sources that do not support
+    // marketable flag, we need to apply the filtering at the end
+    const artworks = args.marketable
+      ? await getMarketableArtworks(
```

Comment:
**question**: Bummer that we need to re-fetch the artworks here. Do we have a Jira ticket for supporting a similar `marketable` query param at least on the non-fallback artworks API endpoint in Gravity?